onnx/defs/tensor/defs.cc

```diff
@@ -2520,8 +2564,9 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Input(
             3,
             "sizes",
-            "The size of the output tensor. The number of elements of 'sizes' should be the same as the"
-            " rank of input 'X'. Only one of 'scales' and 'sizes' can be specified.",
+            "Target size of the output tensor. Its interpretation depends on the 'keep_aspect_ratio_policy' value."
```

Comment:
I assume that the aspect-ratio-policy applies whether scales or sizes is specified?

Reply:
In my opinion, `aspect-ratio-policy` doesn't make sense for scales. If you intend to keep aspect ratio then you set the scale to the same value for all the relevant dimensions. 
Example: 
- Input (300, 200, 3)
You want to double the height, while keeping aspect ratio between height and width.
- Scales (2, 2, 1)

Reply:
One minor consideration might be to choose whichever makes implementations (of the op) simpler, but it is probably not a significant difference in this case. So, I am fine with your choice, but it would help avoid any confusion/ambiguity if we say this explicitly.